src/panels/lovelace/components/hui-image.js

```diff
@@ -95,31 +92,49 @@ class HuiImage extends LocalizeMixin(PolymerElement) {
       _imageSrc: null,
       _error: true
     });
+    this.$.image.classList.add('hidden');
```

Comment:
Should we hide it or render a broken image?

We could have a CSS like this:

```css
background: grey url(https://thewindowsclub-thewindowsclubco.netdna-ssl.com/wp-content/uploads/2018/06/Broken-image-icon-in-Chrome.gif) center/30px no-repeat
```

(we should obviously include the image in our repo)

Reply:
Also, height should be the height of the last loaded image if available.

Reply:
:+1: I like this. I'm at work, but I'll try to get an update in this morning